lib/runner.coffee

```diff
@@ -0,0 +1,89 @@
+{Emitter, BufferedProcess} = require 'atom'
```

Comment:
@rgbkrk I just did a spike on that and I feel it'll either complicate things or make them less extensible.

Lets say I only use the `child_process` API. Then when I want to have more than one Formatter, Runner should either 

* be aware that it can call multiple callback functions (eg. one `stdoutFunc` per each Formatter)
* implement custom event system (Emitter does it for me)

I'd personally stick with the current implementation unless you had a bigger idea in mind (I have rather zero-ish experience in both Atom and Node APIs)